libfaketime/__init__.py

```diff
@@ -181,4 +176,72 @@ def __exit__(self, *exc):
     start = __enter__
     stop = __exit__
 
+    # Decorator-style use support
```

Comment:
Is this copied out of freezegun? I'm fine with that, but it'd be good to link them in case they diverge later on.